modules/nodes-pods-configuring-run-once.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-configuring.adoc
+
+[id='nodes-pods-configuring-run-once_{context}']
+= Limiting the duration of run-once pods
+
+{product-title} relies on run-once pods to perform tasks such as deploying a pod
+or performing a build. Run-once pods are pods that have a `RestartPolicy` of
+`Never` or `OnFailure`.
+
+The cluster administrator can use the *RunOnceDuration* admission control
```

Comment:
Wrong format

Reply:
We need to get consistent on this stuff!

Reply:
We do! I think this was one of the formatting bits that changed in 4.0.